docs/usermanual.md

```diff
@@ -116,10 +116,37 @@ tools available online like [this one](https://string-functions.com/string-hex.a
 
 It is possible report as a measure a NGSI-v2 or NGSI-LD payload when related device/group is configured with
 `payloadType` `ngsiv2` or `ngsild`. In these cases payload is ingested as measure where entity attributes are measure
-attributes and `id` and `type` are ignored, since `id` and `type` from device/group configuration provisioned are used,
-as well as `actionType`.
+attributes. The entity ID and type are also include as attributes `measure_id` and `measure_type` as described 
+[here](https://github.dev/telefonicaid/iotagent-node-lib/doc/api.md#special-measures-and-attributes-names) (both using 
+attribute type `string`(**FIXME [#810](https://github.com/telefonicaid/iotagent-json/issues/810)**: The type is going to be changed to `Text` instead of `string` )). Destination entity ID and entity type is taken from device/group configuration or provision,
```

Comment:
```suggestion
attribute type `string` (**FIXME [#810](https://github.com/telefonicaid/iotagent-json/issues/810)**: The type is going to be changed to `Text` instead of `string` , fix also examples above). Destination entity ID and entity type is taken from device/group configuration or provision,
```